src/element.ts

```diff
@@ -64,7 +64,7 @@ export class AtomTerminal extends HTMLElement {
       const intersectionObserver = new IntersectionObserver(
         async (entries) => {
           const lastEntry = entries.pop()
-          if (lastEntry && lastEntry.intersectionRatio === 1.0) {
+          if (lastEntry && lastEntry.intersectionRatio > 0.0) {
```

Comment:
Should we use `isIntersecting` property?

```suggestion
          if (lastEntry && lastEntry.isIntersecting) {
```

Reply:
I tried it does not work. It seems backward! 
https://github.com/atom-ide-community/terminal/pull/11#issuecomment-690858111